src/ui/server/controllers/insurance/eligibility/buyer-country/countries.test.ts

```diff
@@ -0,0 +1,187 @@
+import { post } from '.';
+import { PAGES } from '../../../../content-strings';
+import { FIELD_IDS } from '../../../../constants';
+import { INSURANCE_ROUTES } from '../../../../constants/routes/insurance';
+import api from '../../../../api';
+import getCountryByIsoCode from '../../../../helpers/get-country-by-iso-code';
+import mapSubmittedEligibilityCountry from '../../../../helpers/mappings/map-submitted-eligibility-country';
+import { updateSubmittedData } from '../../../../helpers/update-submitted-data/insurance';
+import { Country, Request, Response } from '../../../../../types';
+import { mockReq, mockRes, mockCountries } from '../../../../test-mocks';
+
+const {
+  ELIGIBILITY: { CANNOT_APPLY: CANNOT_APPLY_ROUTE, TOTAL_VALUE_INSURED, BUYER_COUNTRY_CHANGE, CHECK_YOUR_ANSWERS, CONTRACT_TOO_SHORT },
+} = INSURANCE_ROUTES;
+
+describe('controllers/insurance/eligibility/buyer-country', () => {
```

Comment:
Should we rename this file`country-redirects`?

And add ` - redirects` to the `describe` here?